qa/tasks/keycloak.py

```diff
@@ -207,6 +208,93 @@ def run_admin_cmds(ctx,config):
         ans3= 'client_secret={}'.format(out2[15:51])
         clientid='client_id={}'.format(client_name)
 
+        proto_map = pre1+"/protocol-mappers/models"
+        uname = "username=testuser"
+        upass = "password=testuser"
+
+        remote.run(
+            args=[
+                '{tdir}/bin/kcadm.sh'.format(tdir=get_keycloak_dir(ctx,config)),
+                'create', 'users',
+                '-s', uname,
+                '-s', 'enabled=true',
+                '-s', 'attributes.\"https://aws.amazon.com/tags\"=\"{"principal_tags":{"Department":["Engineering", "Marketing"]}}\"',
+                '-r', realm_name, 
+            ],
+        )
+
+        sample = 'testuser'
+
+        remote.run(
+            args=[
+                '{tdir}/bin/kcadm.sh'.format(tdir=get_keycloak_dir(ctx,config)),
+                'set-password',
+                '-r', realm_name,
+                '--username', sample,
+                '--new-password', sample,
+            ],
+        )
+
+        proto_data = "{\n"
+        proto_data += '"protocol":"openid-connect",\n'
+        proto_data += '"name": "https://aws.amazon.com/tags",\n'
+        proto_data += '"protocolMapper": "oidc-usermodel-attribute-mapper",\n'
+        proto_data += '"config": {\n'
+        proto_data += '"user.attribute":"https://aws.amazon.com/tags",\n'
+        proto_data += '"claim.name":"https://aws.amazon.com/tags",\n'
```

Comment:
You have to escape the '.' in aws.amazon.com, else it will be an issue later when we parse the token in rgw